RAW_RELEASE_NOTES.md

```diff
@@ -12,3 +12,16 @@ final version.
 * Added transport socket interface to allow custom implementation of transport socket. A transport socket
   provides read and write logic with buffer encryption and decryption. The exising TLS implementation is
   refactored with the interface.
+
+=============================================================
+
+Added supporting the priority field in eds LocalityLbEndpoints for several types of
```

Comment:
Nit: s/eds/EDS, add Sphinx RST links to LocalityLbEndpoints optionally. Also, how come the ======== formatting vs. the normal bullet points that we've traditionally been using?

Reply:
Simplified per the one line * format.  Should be good for another look at your leisure